changes_test.go

```diff
@@ -2171,6 +2323,30 @@ func (s *changesSuite) TestSimpleBundleEmptyModel(c *gc.C) {
 	s.checkBundle(c, bundleContent, expectedChanges)
 }
 
+func (s *changesSuite) TestKubernetesBundleEmptyModel(c *gc.C) {
+	bundleContent := `
+                bundle: kubernetes
+                applications:
+                    django:
+                        charm: cs:django-4
+                        expose: yes
+                        num_units: 1
+                        options:
+                            key-1: value-1
+                            key-2: value-2
+                        annotations:
+                            gui-x: "10"
+                            gui-y: "50"
+            `
+	expectedChanges := []string{
+		"upload charm cs:django-4 for series kubernetes",
+		"deploy application django with 1 unit on kubernetes using cs:django-4",
```

Comment:
should use two apps, one with scale 2 or more to test output

Reply:
done